src/apps/LandingRegister/Footer.react.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+export default React.createClass({
+  displayName: 'Footer',
+
+  getStyles() {
+    return {
+      footerDiv: {
+        display: 'flex',
```

Comment:
if you would like to check if Radium is necessary go to: http://caniuse.com/#search=flexbox
it seems that support for flexbox has been increased in last time, but IE still needs prefixes